modules/thermal_hydraulics/test/tests/components/heat_transfer_from_heat_structure_1phase/phy.T_wall_transfer_3eqn_z.i

```diff
@@ -51,6 +51,7 @@
     orientation = '0 0 1'
     length = 1
     n_elems = 50
+    scaling_factor_1phase = '1 1 1e-1'
 
     A   = 9.6858407346e-01
```

Comment:
@lindsayad check this out, no formatting :D

Reply:
haha beautiful save!